native/cocos/scene/LODGroup.cpp

```diff
@@ -41,6 +41,14 @@ LODGroup::LODGroup() = default;
 
 LODGroup::~LODGroup() = default;
 
+void LODGroup::attachToScene(RenderScene *scene) { 
+    _scene = scene;
+    // node's active maybe changed dynamically
+    if (!_vecLockedLevels.empty()) {
+        _isLockLevelChanged = true;
```

Comment:
I think should modify `_isLockLevelChanged ` when  `_vecLockedLevels ` is changed, not here. After attaching to scene, it may change too.

Reply:
"forceLodLevels" has already been checked for changes, but in order to reduce the recalculation of LOD every frame, the "resetLockChangeFlag" interface is provided externally. Therefore, when calling "attachScene", it is necessary to recheck whether a certain level of LOD is being forced to be used.